RealmSwift/SwiftUI.swift

```diff
@@ -265,6 +265,55 @@ private class ObservableStorage<ObservedType>: ObservableObject where ObservedTy
     }
 }
 
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+private class ObservableResultsStorage<T>: ObservableStorage<T> where T: RealmSubscribable & ThreadConfined & Equatable {
+    var setupHasRun = false
+    func didSet() {
+        if setupHasRun {
+            setupValue()
+            setupHasRun = true
+        }
+    }
+
+    func setupValue() {
+        fatalError()
+    }
+
+    var sortDescriptor: SortDescriptor? {
```

Comment:
Maybe it is too much work, but now that we are unifying storage for @ObservedResults and @ObservedSectionResults, there seems to be a request for extending @ObservedResults API to allow more than one sortDescriptor, https://github.com/realm/realm-swift/issues/7709,

Reply:
It looks easy to implement but not so entirely trivial that I might as well just do it here.

Reply:
@tgoyne If you say you'll implement it here?, I'll wait for the final review

Reply:
No, I won't be implementing it here. I put a few too many conditionals in that sentence.